### DISCUSSION TITLE
Adding support for markdown output? (plugin)

### DISCUSSION THREAD

**Discussion by mandrasch** [craftcms/ckeditor#240] [Q&A]
Hey,

this is not high priority for us, we're just curious because a developer asked if we could output markdown for the ckeditor fields in a headless setup.

I found that this is possible via a plugin in CkEditor
https://ckeditor.com/docs/ckeditor5/latest/features/markdown.html#installation

I read the docs https://github.com/craftcms/ckeditor/tree/7a456915fe885965c3eaa011d84402db3bdd6084?tab=readme-ov-file#adding-ckeditor-plugins and ...

... I found the plugin here, but there is no `build/` directory in it
https://github.com/ckeditor/ckeditor5/tree/master/packages/ckeditor5-markdown-gfm

Is it therefore safe to assume that adding this would require some hours of work to get it up and running?

Thanks very much for any hints!

**Comment by i-just** [comment]
Hi, have a look at this issue: https://github.com/craftcms/ckeditor/issues/140.

> I found the plugin here, but there is no build/ directory in it

This is a good explanation of how to achieve this: https://github.com/craftcms/ckeditor/issues/119#issuecomment-1651513615
